lib/remote/httprequest.cpp

```diff
@@ -215,7 +215,8 @@ void HttpRequest::FinishHeaders()
 			m_Stream->Write(header.CStr(), header.GetLength());
 		}
 
-		m_Stream->Write("\n", 1);
+		/* Ensure that we use two new lines here, https://www.w3.org/Protocols/rfc2616/rfc2616-sec5.html */
+		m_Stream->Write("\r\n\r\n", 1);
```

Comment:
That would still only send one extra new-line (length: 1). However, I don't think that change is necessary because we already have two new-lines:

1. One from the last header (or the status line in the unlikely event that we don't have any headers).
2. One additional new-line from _this_ line.

Reply:
The weird thing is, that this combination works. I know that the loop before already sends `\n` ... I need to test a little further.

Reply:
... and of course [also the chunked encoding](https://en.wikipedia.org/wiki/Chunked_transfer_encoding#Example).

Reply:
Seems nscp violates the RFC here by just accepting `\r` but not `\r\n` or `\n`.

```
+               //m_Stream->Write("\r\n", 2);
+               m_Stream->Write("\r", 1);
+               //m_Stream->Write("\n", 1);
```

I've tested now 0.5.0.62 and 0.5.2.39 (https://github.com/mickem/nscp/releases/tag/0.5.2.39) as latest stable release. No difference.

 @mickem do you know a little more about this?

Reply:
And it responds w/ "HTTP/1.1 200", not "HTTP/1.1 200 OK".